tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -256,17 +256,10 @@ where
             }
         };
 
-        write!(
-            writer,
-            "{} {}{}: ",
-            fmt_level,
-            full_ctx,
-            if self.display_target {
-                meta.target()
-            } else {
-                ""
-            }
-        )?;
+        write!(writer, "{} {}", fmt_level, full_ctx,)?;
```

Comment:
nit: unnecessary trailing comma here?
```suggestion
        write!(writer, "{} {}", fmt_level, full_ctx)?;
```